test/ci_integration.sh

```diff
@@ -46,6 +47,10 @@ setup_environment() {
   # Terraform input variables
   export TF_VAR_project_id="${PROJECT_ID}"
   export TF_VAR_region="${REGION:-us-east4}"
+
+  # Unique ID used for resource names
+  uuid="$(</proc/sys/kernel/random/uuid)"
+  export UNIQUE_ID="${uuid:0:8}"
```

Comment:
Where is this being used? Grepping through the project source doesn't show anything.

Reply:
Good catch, removed.

```diff
commit 22dbee3d476ddaf4bca5b1b5e898e8df9cb796af
Author: Jeff McCune <jeff@openinfrastructure.co>
Date:   Fri Feb 15 13:33:25 2019 -0800

    Fixup remove unused UNIQUE_ID var

diff --git a/test/ci_integration.sh b/test/ci_integration.sh
index 0ef3f4a..21fbb58 100755
--- a/test/ci_integration.sh
+++ b/test/ci_integration.sh
@@ -47,10 +47,6 @@ setup_environment() {
   # Terraform input variables
   export TF_VAR_project_id="${PROJECT_ID}"
   export TF_VAR_region="${REGION:-us-east4}"
-
-  # Unique ID used for resource names
-  uuid="$(</proc/sys/kernel/random/uuid)"
-  export UNIQUE_ID="${uuid:0:8}"
 }
 
 main() {
```